pkg/pod/pod.go

```diff
@@ -119,8 +119,26 @@ func MakePod(images pipeline.Images, taskRun *v1alpha1.TaskRun, taskSpec v1alpha
 	initContainers = append(initContainers, entrypointInit)
 	volumes = append(volumes, toolsVolume, downwardVolume)
 
+	// Get LimitRange by name if present in ConfigMap tekton-limitrange-config.
+	// Otherwise, pass empty request to resolveResourceRequests.
+	limitRangeName := ""
+	limitRangeConfig, err := kubeclient.CoreV1().ConfigMaps(taskRun.Namespace).Get("tekton-limitrange-config", metav1.GetOptions{})
+	if err == nil {
+		limitRangeName = limitRangeConfig.Data["default-limitrange-name"]
+		if limitRangeName == "" {
+			return nil, fmt.Errorf("tekton-limitrange-config is present but default-limitrange-name key is missing")
+		}
+	}
+	limitRange := &corev1.LimitRange{}
```

Comment:
suggest leaving this as nil by default.  we won't need to compare against the empty limit range in resolveResourceRequests if we do that since we can just assume that if the limitRange is non nil then we should use its values.